docker-compose.yml

```diff
@@ -35,6 +35,7 @@ services:
      FLASK_ENV: "${FLASK_ENV:-development}"
    volumes:
      - ./OpenOversight/:/usr/src/app/OpenOversight/:z
+     - ./service_account_key.json:/usr/src/app/service_account_key.json
```

Comment:
Need to add the `service_account_key.json` file here so it can be accessed by the application.

Reply:
I think you can add the same line to https://github.com/lucyparsons/OpenOversight/blob/develop/docker-compose.prod-img.yml that seems like an okay place for that file to live in prod and staging